notes/semester_02_2019/4-stewardship-2019-08-06.md

```diff
@@ -0,0 +1,352 @@
+    - Think that's part of sociological relationship to data. Hardest aspect to remedy. Requires us to stop everybody, Hey! You were upset by CA (Cambridge Analytica) and somebody aggregating your information in ways you disagree with
+    - Many remedies will involve convos we didn't want to have before. How do we negotiate that?
+    - Shifting labor is real, moving work onto people who weren't doing work before. When thinking about stewardship of information. That's really what we're talking about, when decentralizing - pushing the burden of keeping thing available and online, turning thin clients into stewards.
+    - Paul Frazee's pick think or thin client. New responsibility, hard to change rules of game now. When internet still works this way, everyone's freeloading. Treating data people are creating as labor that can be monetized
+    - Feels like central pain of decentralization and feels like one of stewardship.
+    - Became a steward overnight and they didn't realize it. How do we have that convo in meaningful way, seeing value in taking SSB road
+- RB: Couple thoughts in line and counter
+    - First immediate reaction, pushing concept of stewardship into protocol. Good from perspective of: it makes it easy to do. Worry about ways it could hide it from conscious consideration
+    - Counterpoint in here is part of what's important, in equitable society, something we should be thinking about. A new point of consideration in these new technologies. Big part of what's different in distributed web stuff. Where things live is something you should be thinking about now, it only exists if it's well stewarded by multiple people. Bears real thinking about
+    - To some degree, think it's critical in systems we want to work well that gets surfaced somehow
+    - Can't criticize SSB, keep meaning to read up on protocol. To degree of declaring peer/friend, you're stewarding their data. Good thing, makes it a conscious decision. But if you're doing it also for the people you're around maybe it's less good
+    - Counterpoint - I think concerning issue isn't so much scale, as it is scope. Part about scale is more about measurate(?) scale. Mauve's point - can't expect people to also be stewards of LHC, but institutions have same sort of needs of co-work and co-stewardship. The instituions that support that also have to support scale
+    - The concerning part to pay attention to is  scope. Intentional communities where we can consciously think about what we're stewarding and who we're communicating with. 
+    - The problem is Google storing 5 billion users data instead of community of 1000 or 10, but maybe it's lots of data. It's fine because it's a scope we can make meaningful
```

Comment:
```suggestion
    - The problem is Google storing 5 billion users' data instead of community of 1000 or 10, but maybe it's lots of data. It's fine because it's a scope we can make meaningful
```